core/src/main/resources/application.yml

```diff
@@ -106,6 +106,7 @@ altus:
 
 cb:
   enabledplatforms: AZURE,AWS,GCP
+  enabledgovplatforms: AWS
```

Comment:
just a side note: you will need a counterpart of this in many places like cbd, CSI etc.

Reply:
I know 👯